README.md

```diff
@@ -9,22 +11,38 @@ Issues and feature requests posted to GitHub are not stored in any
 publicly available location, so I find that I quickly lose track of
 the things that I've sent them.
 
+## For GitHubbers
+
 If you are a GitHubber, and you would like to respond and close issues
 on this repository, please let me (or one of the other collaborators)
 know, and we'll add you as a collaborator.  Or maybe use some secret
 GitHub internal APIs to get access, if you have such things.  If you
 want to know what people are asking for, or ask for them to provide
 details about use cases etc, then head over to the issues list.
 
-Post an issue if you have an issue or feature request for GitHub.
-**But you should also [contact GitHub](https://support.github.com/contact), since this repo is
-strictly for our own (unofficial) tracking purposes.** If they reply,
-(and they usually do, quickly) and if it is not a confidential matter
-like a security disclosure, add their reply to the issue so that other
-users can know what their official response was.
+## If you have an issue or feature request for GitHub
+
+1. Search for existing [issues](https://github.com/isaacs/github/issues) here
+2. If you did not find any existing issue for your topic, post a new issue 
+3. **But you should also email support@github.com, since this repo is strictly for our own (unofficial) tracking purposes.** 
```

Comment:
Don't start this sentence with but _or_ add it as an additional step. Instead, add it as a blockquote like so:

> Additionally, please email support@github.com because this repository is strictly for our own (unofficial) tracking purposes.